server/src/inmantals/tcpserver.py

```diff
@@ -35,7 +39,12 @@ def main():
     logging.basicConfig(level=logging.DEBUG)
 
     server = JsonRpcServer(InmantaLSHandler)
-    server.listen(5432)
+    port = int(sys.argv[1]) if len(sys.argv) > 1 else 5432
+    sys.stdout.write(f"starting server on port {port}")
+    sys.stdout.flush()
+    sys.stderr.write(f"Log file can be found at {logfile}")
+    sys.stderr.flush()
```

Comment:
I'm not sure we should log this to stderr, stdout seems like the conventional stream for a message like this. Any specific reason to log to stderr?

Reply:
Currently, only the stderr is displayed to the user in the output channel in vscode (with both the pipe and tcp server), that's why I've logged it there. But I agree that it would be better in stdout. Do you think we should also show the stdout in the output channel?

Reply:
I'm not sure. I think it would be ok to show stdout as well. @arnaudsjs what do you think?

Reply:
I don't have a strong opinion on this, but I agree that stdout is more conventional. 

What is more strange to me is the fact that the first log entry is logged to stdout and the second to stderr?

Reply:
Right, I changed it so both use stdout